pkg/sdk/testint/tags_integration_test.go

```diff
@@ -281,3 +283,915 @@ func TestInt_TagsShowByID(t *testing.T) {
+			setupObject: func() sdk.SchemaObjectIdentifierWithArguments {
+				// cleanup is set up in the Create function
+				function := testClientHelper().Function.Create(t, sdk.DataTypeInt)
+				return function.ID()
+			},
+			setTags: func(id sdk.SchemaObjectIdentifierWithArguments, tags []sdk.TagAssociation) error {
+				return client.Functions.Alter(ctx, sdk.NewAlterFunctionRequest(id).WithSetTags(tags))
+			},
+			unsetTags: func(id sdk.SchemaObjectIdentifierWithArguments, tags []sdk.ObjectIdentifier) error {
+				return client.Functions.Alter(ctx, sdk.NewAlterFunctionRequest(id).WithUnsetTags(tags))
+			},
+		},
+		{
+			name:       "ExternalFunction",
+			objectType: sdk.ObjectTypeExternalFunction,
+			setupObject: func() sdk.SchemaObjectIdentifierWithArguments {
+				integration, integrationCleanup := testClientHelper().ApiIntegration.CreateApiIntegration(t)
+				t.Cleanup(integrationCleanup)
+				// cleanup is set up in the Create function
+				function := testClientHelper().ExternalFunction.Create(t, integration.ID(), sdk.DataTypeInt)
+				return function.ID()
+			},
+			setTags: func(id sdk.SchemaObjectIdentifierWithArguments, tags []sdk.TagAssociation) error {
+				return client.Functions.Alter(ctx, sdk.NewAlterFunctionRequest(id).WithSetTags(tags))
+			},
+			unsetTags: func(id sdk.SchemaObjectIdentifierWithArguments, tags []sdk.ObjectIdentifier) error {
+				return client.Functions.Alter(ctx, sdk.NewAlterFunctionRequest(id).WithUnsetTags(tags))
+			},
+		},
+		{
+			name:       "Procedure",
+			objectType: sdk.ObjectTypeProcedure,
+			setupObject: func() sdk.SchemaObjectIdentifierWithArguments {
+				// cleanup is set up in the Create procedure
+				procedure := testClientHelper().Procedure.Create(t, sdk.DataTypeInt)
+				return procedure.ID()
+			},
+			setTags: func(id sdk.SchemaObjectIdentifierWithArguments, tags []sdk.TagAssociation) error {
+				return client.Procedures.Alter(ctx, sdk.NewAlterProcedureRequest(id).WithSetTags(tags))
+			},
+			unsetTags: func(id sdk.SchemaObjectIdentifierWithArguments, tags []sdk.ObjectIdentifier) error {
+				return client.Procedures.Alter(ctx, sdk.NewAlterProcedureRequest(id).WithUnsetTags(tags))
+			},
+		},
+	}
+
+	for _, tc := range schemaObjectWithArgumentsTestCases {
+		t.Run(fmt.Sprintf("schema object with arguments %s", tc.name), func(t *testing.T) {
+			id := tc.setupObject()
+			err := tc.setTags(id, tags)
+			require.NoError(t, err)
+
+			returnedTagValue, err := client.SystemFunctions.GetTag(ctx, tag.ID(), id, tc.objectType)
+			require.NoError(t, err)
+			assert.Equal(t, tagValue, returnedTagValue)
+
+			err = tc.unsetTags(id, unsetTags)
+			require.NoError(t, err)
+
+			_, err = client.SystemFunctions.GetTag(ctx, tag.ID(), id, tc.objectType)
+			require.ErrorContains(t, err, "sql: Scan error on column index 0, name \"TAG\": converting NULL to string is unsupported")
+		})
+	}
+}
+
+func createDatabaseFromShare(t *testing.T) (*sdk.Database, func()) {
```

Comment:
let's use the methods present in the client helper or add if missing (this is not the first case where we set up such database)